lib/simp/cli/lib/track_output.rb

```diff
@@ -0,0 +1,94 @@
+require 'pty'
```

Comment:
lib directory under lib/simp/cli is something I have wanted to remove.  Instead of adding to this odd directory, please create a /lib/simp/cli/bootstrap directory for this file.